money/bostonhousing.qmd

```diff
@@ -0,0 +1,88 @@
+categories:
+  - linear regression
+  - EDA
+  - classification
+data:
+  year: 1978
+  files: HousingData.csv
+---
+
+## Motivation
+
+Harrison and Rubinfeld studied families' willingness to pay for clean air using Boston area housing market data in 1978. To consider a more local measure of air pollution and a variety of factors influencing housing prices, the authors developed a dataset using a mixture of governmental data and prior research articles. The data was collected on the census tract level, using the nitric oxides (NOX) concentration as a proxy for air quality, the median housing value of owner-occupied homes in a census tract, as well as other control variables like accessibility to radial highways and weighted distance to employment centers. The authors use a hedonic pricing approach, which assumes that the price of a good or service can be modeled as a function of features both internal and external to the good or service. The authors found a marginal air pollution damages (as revealed in the housing market) increases with the level of air pollution and with household income.
+
+This dataset was included as one of ``scikit-learn’s`` and tensorflow’s standard toy datasets, speficially for housing price predictions. It has also been used as a benchmark tool for many machine learning papers. In 2020, users brought the dataset’s fairness issues to the scikit-learn development team (see scikit-learn issue [#16155](https://github.com/scikit-learn/scikit-learn/issues/16155)), after which the team decided to remove the dataset in scikit-learn version 1.2. The discussion on it points out that first, race/ethnicity proportion should not be a factor in housing price prediction tasks, and second, the parabolic form of the `B` variable assumes that there is an "ideal" proportion of Black residents, which is highly problematic.
+
+Since then, fairness practitioners have used this dataset to demonstrate what fairness issues in machine learning could look like and how to assess them. This data project is inspired by the [Fairlearn project](https://fairlearn.org/v0.12/user_guide/datasets/boston_housing_data.html).
+
+## Data
+
+The data consists of 506 census tracts in Boston, collected from a mixture of surveys, administrative records, and other research papers. All variables related to houses are only considering owner-occupied homes (eg. median value).
+
+### Data preview
+
+```{r, echo=FALSE, results="asis"}
+source("../preview_dataset.R")
+preview_datasets()
+```
+
+### Variable descriptions
+
+| Variable | Description |
+|----|-------------|
+| CRIM | per capita crime rate by town |

```

Comment:
Are the "by town" variables not by Census tract? So some of these are from Census and some from towns that span multiple tracts?

If so, up in Data where you say it comes from census tracts, you might add a sentence or two: "Some variables are recorded at the census tract level, and some come from the town each tract is in. One town may contain many census tracts."